integration/js/script/run-test

```diff
@@ -142,41 +142,57 @@ case $1 in
+         r configs/browserCompatibilityTest/desktop/meeting_leave_video_test.config.json
+         r configs/browserCompatibilityTest/desktop/reconnection_test.config.json
          ;;
       content_share)
-         r configs/browserCompatibilityTest/content_share_screen_capture_test.config.json
-         r configs/browserCompatibilityTest/meeting_leave_content_share_test.config.json
-         r configs/browserCompatibilityTest/content_share_video_test.config.json
-         r configs/browserCompatibilityTest/content_share_only_allow_two_test.config.json
-         r configs/browserCompatibilityTest/content_share_join_later_test.config.json
-         r configs/browserCompatibilityTest/app_quit_content_share_test.config.json
+         r configs/browserCompatibilityTest/desktop/content_share_screen_capture_test.config.json
+         r configs/browserCompatibilityTest/desktop/meeting_leave_content_share_test.config.json
+         r configs/browserCompatibilityTest/desktop/content_share_video_test.config.json
+         r configs/browserCompatibilityTest/desktop/content_share_only_allow_two_test.config.json
+         r configs/browserCompatibilityTest/desktop/content_share_join_later_test.config.json
+         r configs/browserCompatibilityTest/desktop/app_quit_content_share_test.config.json
          ;;
       data_message)
-         r configs/browserCompatibilityTest/data_message_test.config.json
+         r configs/browserCompatibilityTest/desktop/data_message_test.config.json
          ;;
       meeting_readiness_checker)
-         r configs/browserCompatibilityTest/meeting_readiness_checker_test.config.json
+         r configs/browserCompatibilityTest/desktop/meeting_readiness_checker_test.config.json
          ;;
       background_blur)
-         r configs/browserCompatibilityTest/background_blur_test.config.json
+         r configs/browserCompatibilityTest/desktop/background_blur_test.config.json
          ;;
       voice_focus)
-         r configs/browserCompatibilityTest/voice_focus_test.config.json
+         r configs/browserCompatibilityTest/desktop/voice_focus_test.config.json
+         ;;
+      linux_audio)
+         r configs/browserCompatibilityTest/linux/linux_audio_test.config.json
+         ;;
+      linux_video)
+         r configs/browserCompatibilityTest/linux/linux_video_test.config.json
+         ;;
+      linux_content_share)
+         r configs/browserCompatibilityTest/linux/linux_content_share_join_later_test.config.json
+         r configs/browserCompatibilityTest/linux/linux_content_share_only_allow_two_test.config.json
+         r configs/browserCompatibilityTest/linux/linux_content_share_screen_capture_test.config.json
+         ;;
+      linux_other)
+         r configs/browserCompatibilityTest/linux/linux_data_message_test.config.json
+         r configs/browserCompatibilityTest/linux/linux_meeting_end_test.config.json
+         r configs/browserCompatibilityTest/linux/linux_sip_call_test.config.json
```

Comment:
Could you remove the `linux_` prefix for all these config fiels, since we already have the `linux` folder?

Reply:
Done